submit-api/src/submit_api/services/package.py

```diff
@@ -342,6 +342,23 @@ def _resubmit_package(cls, package, session):
         cls._log_activity_submission(package, ActivityActionType.UPDATED_SUBMISSION.value, session)
         return package
 
+    @classmethod
+    def accept_update_request(cls, package_id):
```

Comment:
have you thought about updating the active attribute to false when we accept an udpate request

Reply:
Sure let me check on that.

Reply:
Updated to set the attribute to false